tools/RAiDER/downloadGNSSDelays.py

```diff
@@ -80,6 +81,11 @@ def create_parser():
         help='Physically download data. Note this option is not necessary to proceed with statistical analyses, as data can be handled virtually in the program.',
         action='store_true', dest='download', default=False)
 
+    misc.add_argument(
+        '--cpus',
+        help='Specify number of cpus to be used for multiprocessing. May specify "all" at your own discretion.',
+        dest='numCPUs', type=parse_cpus, default=4)
+
```

Comment:
@Askaholic how are number of cpu's controlled on the RAiDER size?

Reply:
They are computed based on the number of chunks.